addons/danger/functions/fnc_vehicleAssault.sqf

```diff
@@ -65,8 +66,11 @@ _pos = (eyePos _unit) vectorAdd ((eyePos _unit vectorFromTo (AGLToASL (selectRan
 // look at position
 _vehicle doWatch ASLtoAGL _pos;
 
+// recheck
+if (_vehicle distance (ASLToAGL _pos) < GVAR(minSuppression_range)) exitWith {false};
```

Comment:
in L34 you use distance2D and here distance, is that wanted?

Reply:
Fixed